docs/content/v0.20/architecture.md

```diff
@@ -239,20 +240,20 @@ as part of the decoupling of transport from binding as described above, the foll
   and one client (or more in the future) to connect to one or more ITS shards.
   - The OCM-connected client is used to copy/update/remove the wrapped objects
     into/from the OCM mailbox namespaces.
+- Status controller - TODO 
 
-so after the described change, we will have three controllers in the KubeStellar controller
-manager: the binding controller, the transport controller and the status controllor.
 
 ### Binding Controller
 
 The Binding controller is responsible for watching workload objects and 
-`BindingPolicy` objects, and wrapping and delivering objects to the ITS
-based on binding policies. As a step towards the planned change that is described in this document, Binding controller also creates for each `BindingPolicy` a matching `Binding` object in the WDS. A `Binding` object is mapped 1:1 to a `BindingPolicy` object and contains references to the concrete list of workload objects and references to the concrete list of destinations that were selected by the policy and should get all the workload objects.
-Until the planned change is completed, nothing besides this controller touches the `Binding` objects. 
+`BindingPolicy` objects, and maintains for each of the latter a matching `Binding` object in the WDS. 
+A `Binding` object is mapped 1:1 to a `BindingPolicy` object and contains references to the concrete list of workload 
+objects and references to the concrete list of destinations that were selected by the policy.
+
 
-The architecture and the event flow for **CURRENT** code for create/update object events is
+The architecture and the event flow of the code for create/update object events is
 illustrated in Figure 3 (some details might be omitted to make the flow easier
-to understand). Once the planned change that is describe in this document is integrated we will update this figure accordingly.
+to understand). 
 
 <figure>
```

Comment:
FYI: this PR did not introduce this problem, but carried on with it. This figure depicts only  one of the "flows" in the binding controller, the one responding to a notification about a workload object. There are others.

Reply:
Lets only do mandatory changes in this PR, and perhaps rework the architecture docs after.